packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -231,28 +233,41 @@ export function convertRuntimeToPlugin(
         }
       }
 
+      const tracedFiles: {
+        absolutePath: string;
+        relativePath: string;
+      }[] = [];
+
+      const linkers = Object.entries(lambdaFiles).map(
+        async ([relPath, file]) => {
+          const newPath = join(traceDir, relPath);
+
+          // The handler was already moved into position above.
+          if (relPath === handlerFileBase) {
+            return;
+          }
+
+          tracedFiles.push({ absolutePath: newPath, relativePath: relPath });
+          const { type } = file;
+
+          if (type === 'FileBlob') {
```

Comment:
What about the other file types such as `FileFsRef` and `FileRef`?

Reply:
Fixed now!